src/components/QuizDetail.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../stylesheet/Quiz.css';
+
+function QuizDetail(props) {
+  const { quiz } = props;
+  return (
+    <div>
+      Details Of Quiz
+      <div className="QuizDetail">
+        <div>
+          <div className="QuizDetailText">
+            Min Limit:
+            {' '}
+            {quiz.minLimitVal}
```

Comment:
Don't need to specify Val behind each variable name.

Reply:
Done